### PR TITLE
Sort option for links

### PR DESCRIPTION
**Pull request by AshantiMutinta**
## testing/benchmarking notes

( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )

## followups

( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )

## changelog

- [x] if this is a code change that effects some consumer (e.g. zome developers) of holochain core,  then it has been added to [our between-release changelog](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG-UNRELEASED.md) with the format 

```markdown
- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)
```

## documentation

- [x] this code has been documented according to our [docs checklist](https://hackmd.io/@freesig/Hk9AmKJNS)

### CODE REVIEW

File: app_spec/zomes/simple/code/src/lib.rs [hunk 1]
```diff
@@ -51,10 +51,11 @@ pub fn handle_delete_my_link_with_tag(base: Address,target : String,tag:String)
 }
 
 
-pub fn handle_get_my_links(agent : Address,status_request:Option<LinksStatusRequestKind>) ->ZomeApiResult<GetLinksResult>
+pub fn handle_get_my_links(agent : Address,status_request:Option<LinksStatusRequestKind>,sort_order:Option<SortOrder>) ->ZomeApiResult<GetLinksResult>
```
Comment (neonphog) [inline app_spec/zomes/simple/code/src/lib.rs]: Looks like you're just following the existing paradigm, so no need to change, but I find this pretty hard to parse without spaces - maybe we should add these rust files to our `fmt` check as a separate code hygiene effort?

Comment (AshantiMutinta) [reply app_spec/zomes/simple/code/src/lib.rs]: I 100% agree! i dont know if it is because they are zomes but we should definitely look into this
